pom.xml

```diff
@@ -146,6 +148,11 @@
             <!-- <version>${jackson.version}</version> -->
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>

```

Comment:
black ducks scan team at GE does not want us to use findBugs as dependency please remove it.